mobile/src/screens/meditation-home/meditation-home.tsx

```diff
@@ -64,7 +53,9 @@ const MeditationHome: React.FC = () => {
             return (
               <Card
                 title={item.name}
-                onPress={handleSelectMeditation}
+                onPress={(): void => {
+                  handleSelectMeditation(item.name);
+                }}
```

Comment:
```suggestion
                onPress={handleSelectMeditation(item.name)}}
```

Lets use it directly, as it's our approach in the project:

```
const handleSelectMeditation = useCallback(
  (title: string) => {
    return () => {
      navigation.navigate(ChatScreenName.MEDITATION_LIST, { title });
    };
  },
  [navigation],
);
```

Reply:
@Dmytro-Kucherenko i cant use it directly, i was said to do it like this, to avoid extra props

Reply:
@iamAlinaaa Won't method with currying functions I described work fine, cause it returns another function? It's our approach in web, correct me if not

Reply:
we should avoid returning function from function, it complicates the logic

Reply:
as i see on frontend we don't return function but we pass title inside the component, which is not a good approach, we will need to fix this